docker/Dockerfile

```diff
@@ -0,0 +1,184 @@
+FROM centos:7
+
+LABEL maintainer="rluckie@cisco.com"
+
+COPY files/yum.repos.d/** /etc/yum.repos.d/
+COPY files/python-requirements/** /tmp/python-requirements/
+COPY files/provision-user /usr/local/bin/provision-user
+COPY files/start-dockerd /usr/local/bin/start-dockerd
+COPY files/awake /usr/local/bin/awake
+
+RUN yum update --nogpgcheck -y && \
+    yum install --nogpgcheck -y epel-release deltarpm yum-utils https://centos7.iuscommunity.org/ius-release.rpm && \
+    yum group install --nogpgcheck -y "Development Tools" && \
+    yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo && \
+    yum install -y \
+        bc \
+        bind \
+        bind-utils \
+        colordiff \
+        ctags \
+        curl-devel \
+        device-mapper-persistent-data \
+        docker-ce \
+        dos2unix \
+        emacs \
```

Comment:
I think we can avoid the X11 libs by: yum install emacs-nox

Reply:
I'll test that out in followup PR.

Reply:
Followup PR is good.